README.md

```diff
@@ -19,16 +19,27 @@ Use it:
 
 You'll get an XML file with your results in it.
 
-## More Permanent Usage
+## Permanent Usage
 
-Add it to your Gemfile if you're using [Bundler][bundler]. Put it in the same groups as rspec.
+Add it to your Gemfile if you're using [Bundler][bundler]. Put it in the same groups as rspec. 
+I use it with the excellent [Fuubar formatter][fuubar].
 
-In your .rspec, usually alongside another formatter, add:
+### Single process testing
 
-    --format RspecJunitFormatter
-    --out rspec.xml
+In your `.rspec`, usually alongside another formatter, add:
 
-I use it with the excellent [Fuubar formatter][fuubar].
+```ruby
+ --format RspecJunitFormatter
+ --out rspec.xml
+ ```
+
+### Parallel tests
+
+For use with `parallel_tests`, be sure to use the `TEST_ENV_NUMBER` in the output file option (in `.rspec` or `.rspec_parallel`) to avoid concurrent process write conflicts.
+```ruby
+--format RspecJunitFormatter
+--out tmp/rspec<%= ENV['TEST_ENV_NUMBER'] %>.xml
```

Comment:
I think mine is setup now as target.
On Oct 21, 2015 1:19 PM, "Benjamin Fleischer" <notifications@github.com>
wrote:

> In README.md
> <https://github.com/sj26/rspec_junit_formatter/pull/34#discussion_r42661791>
> :
>
> >
> > -I use it with the excellent [Fuubar formatter][fuubar].
> > +```ruby
> > + --format RspecJunitFormatter
> > + --out rspec.xml
> > + ```
> > +
> > +### Parallel tests
> > +
> > +For use with `parallel_tests`, be sure to use the `TEST_ENV_NUMBER` in the output file option (in `.rspec` or `.rspec_parallel`) to avoid concurrent process write conflicts.
> > +```ruby
> > +--format RspecJunitFormatter
> > +--out tmp/rspec<%= ENV['TEST_ENV_NUMBER'] %>.xml
>
> why tmp?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sj26/rspec_junit_formatter/pull/34/files#r42661791>.
>